.env.sample

```diff
@@ -1,10 +1,11 @@
-PORT = an integer representing the port through which the server will be accessed e.g. 3000
-SECRET_KEY = A string to be used when generating token, it can be any length.
-DATABASE_URL= postgres://user_name:password@host:port/db_name
-TEST_DATABASE_URL=postgres://user_name:password@host:port/test_db_name
-SENDGRID_API_KEY= api key from sendgrid
-SERVER_MAIL= email address
-google_client=string from google
-google_secret=string from google
-facebook_client=string from facebook
-facebook_secret=string from facebook
\ No newline at end of file
+PORT = AN INTEGER REPRESENTING THE PORT THROUGH WHICH THE SERVER WILL BE ACCESSED E.G. 3000
+SECRET_KEY = A STRING TO BE USED WHEN GENERATING TOKEN, IT CAN BE ANY LENGTH.
+DATABASE_URL = POSTGRES://USER_NAME:PASSWORD@HOST:PORT/DB_NAME
+TEST_DATABASE_URL = POSTGRES://USER_NAME:PASSWORD@HOST:PORT/TEST_DB_NAME
+SENDGRID_API_KEY = API KEY FROM SENDGRID
+SERVER_MAIL = EMAIL ADDRESS
+GOOGLE_CLIENT = STRING FROM GOOGLE
```

Comment:
This has to reflect on the passport.js file (line 14-17).

Reply:
And the capitalization was only expected on the variable name, not the description

Reply:
Sorry about that, i'm fixing it now